R/app.R

```diff
@@ -538,3 +540,31 @@ knit_print.reactive <- function(x, ..., inline = FALSE) {
     x()
   }), inline = inline)
 }
+
+
+
+# throw an informative warning if a www subdirectory conflicts
+# with another resourcePath
+wwwSubDirConflicts <- function(wwwdir) {
+  wwwSubDirs <- list.dirs(wwwDir, recursive = FALSE, full.names = FALSE)
+  resourcePrefixes <- c(
+    names(.globals$resourcePaths),
+    names(.globals$resources),
```

Comment:
`names(.globals$resources)` shouldn't be necessary, since it should have the same content as `names(.globals$resourcePaths)`.